defaults/main.yml

```diff
@@ -1,7 +1,17 @@
 ---
 agent_type: ""
-package_state: "present"
+package_state: present
 
-# default vars for the monitoring agent
+# Default vars for the monitoring agent
 monitoring_package_name: stackdriver-agent
 monitoring_service_name: stackdriver-agent
+
+# Local path to the config file to copy onto the remote server. Can be absolute or relative.
+monitoring_config_file_local: ""
+monitoring_config_file_remote: /etc/stackdriver/test.conf
```

Comment:
Should this be `/etc/stackdriver/collectd.conf`? BTW, is this `monitoring_config_file_remote` var also configurable by the user? If so, we should remind them that the file `monitoring_config_file_remote` has to include a line that includes the directory of `monitoring_plugin_dir_remote`.

Reply:
On a second thought, I think we should just hard code the remote location without allowing users to customize it. Our documentation, troubleshooting and support tooling all have dependencies on the remote location. A customized location may cause unexpected issues. We can revisit it if some customer definitely needs to customize this.

Reply:
Ok, I will hardcode in `/etc/stackdriver/collectd.conf`.